skribl/word_list.txt

```diff
@@ -0,0 +1,13 @@
+zed
+jetson
+point cloud
+aruco tag
+navigation
+state machine
+perception
+percep-memes
+localization
+imu
+kalman filter
```

Comment:
this one might be a little too advanced for new members, might wanna hold off on it for now